### DISCUSSION TITLE
大佬好，演示里使用西游记版本是什么版本，方便分享吗

### DISCUSSION THREAD

**Discussion by mangguo7** [readest/readest#72] [General]
邮箱 mgcola@qq.com，感谢大佬

**Comment by chrox** [comment]
Sorry I cannot share it here. You might need to ask for help in other forums that support epub sharing.

**Comment by mangguo7** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
